template/bash_profile.template.bash

```diff
@@ -26,6 +26,11 @@ export TODO="t"
 # Set this to false to turn off version control status checking within the prompt for all themes
 export SCM_CHECK=true
 
+# Set Xterm/screen/Tmux title with only a short hostname.
+# Comment this (or unset SHORT_HOSTNAME) to fall back on $HOSTNAME.
+
+export SHORT_HOSTNAME=$(hostname -s)
```

Comment:
Since this changes the previous default behavior, wouldn't it make sense to comment this out by default and then enable it on a case-by-case basis?